src/Input/Mouse.cpp

```diff
@@ -125,39 +136,35 @@ void Mouse::_setType(Cursor state)
             break;
         case Cursor::SCROLL_W:
             _ui = new UI::Image("art/intrface/scrwest.frm");
-            _ui->setYOffset( -_ui->height()*0.5);
+            _ui->setOffset(0, -_ui->size().height() / 2);
```

Comment:
Look how old code worked for transition from S to SW. Are you sure new code works correctly?